scripts/linux/nv/gitlab_test_performance.sh

```diff
@@ -40,23 +40,36 @@ echo ">>> Local Server started for further performance test ... >>>"
 export cache_location="cache"
 
 echo ">>> Start performance tests ... >>>"
-heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*ReadNPartitionsFromVersionedLayer/short_test_null_cache" 2>> errors.txt || TEST_FAILURE=1
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*MemoryTest.ReadNPartitionsFromVersionedLayer/short_test_null_cache" 2>> errors.txt || TEST_FAILURE=1
```

Comment:
Maybe it is nicer to create this list automatically based on the --gtest_list_tests output if time allows for such change.

Reply:
Will make sense after we remove the old prefetch test